platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -300,6 +300,18 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)mapView:(MGLMapView *)mapView didDeselectAnnotationView:(MGLAnnotationView *)annotationView;
 
+/**
+ Tells the delegate that one if its annotation views was dragged to a new coordinate.
+ 
+ You should update the coordinate of the associated annotation when this method is called.
```

Comment:
Correct. Next time `-updateAnnotationViews` is called, it will go back to the original location.